Integrations/integration-McAfee-ATD.yml

```diff
@@ -586,53 +582,59 @@ script:
         return uriSuffix;
 
     def detonate(submitType ,sample, timeout, report_type, threshold, fileName):
-        sys.exit(0)
-        result = file_upload(submitType, sample, fileName)
+        result = file_upload(submitType, sample, fileName, None, None, None, None, None, None, None, None)
```

Comment:
Why send all these `None` args?

Reply:
it uses the same uploadfile command of the basic command that has many arguments, and are not needed for the detonate command.
not sending them results a python error.